doc/users/next_whats_new/styles_from_packages.rst

```diff
@@ -0,0 +1,10 @@
+Style files can be imported from third-party packages
+-----------------------------------------------------
+
+Third-party packages can now distribute style files that are globally available
+as follows.  Assume that a package is importable as ``import my.package``, with
+a ``my/package/__init__.py`` module.  Then a
+``my/package/presentation.mplstyle`` style sheet can be used as
+``plt.style.use("my.package.presentation")``.  (Note that the implementation
+does not actually import ``my.package``, making this process safe against
```

Comment:
```suggestion
as follows.  Assume that a package is importable as ``import mypackage``, with
a ``mypackage/__init__.py`` module.  Then a
``mypackage/presentation.mplstyle`` style sheet can be used as
``plt.style.use("mypackage.presentation")``.  (Note that the implementation
does not actually import ``mypackage``, making this process safe against
```
Let's keep this top-level. Even if a package has subpackages, there should almost never be a reason to hide the styles in lower levels.

You may mention the edge-case in yet to be written documentation.